spanner/spanner_snippets/spanner/spanner_opentelemetry_capture_query_stats_metric.go

```diff
@@ -0,0 +1,107 @@
+//go:build go1.20
```

Comment:
please do not use build directives in sample code. they are not visible in the regions displayed to the user, and they require additional maintenance. 
If this sample requires go1.20, create a separate module that requires go1.20 (maybe even in a less redundantly-named directory like `/spanner/opentelemetry` 😄 ).

Reply:
Thanks for the suggestion. I have created a separate module as suggested.